crates/bevy_app/src/lib.rs

```diff
@@ -1,7 +1,35 @@
+use bevy_ecs::StageLabel;
+
 /// The names of the default App stages
-pub mod stage;
+#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
+pub enum CoreStage {
```

Comment:
IMO `CoreStage` and `StartupStage` should be marked #[non_exhaustive] to signal that they're not fixed or exhaustive, and avoid people trying to exhaustively pattern match to them.

Reply:
Why'd they try to match this enum? Values of these types are only produced by user code.

Reply:
Hmm, that's a fair point. I'm trying to think of cases where you might possibly try but can't.